app/screens/interactive_dialog/dialog_element.tsx

```diff
@@ -59,7 +74,7 @@ function DialogElement({
             return;
         }
         onChange(name, newValue);
-    }, [onChange, type, subtype]);
+    }, [type, subtype, onChange, name]);
```

Comment:
for the `parseInt` above, does `newValue as string` work if `newValue` is a `string[]`?

Reply:
The idea is that `string[]` is only getting into this function if we were to accept multiselect selectors (we accept that for Apps but not for Interactive Dialogs). But then the `type` and `subtype` won't be `text` and `number`.